src/libguestfish.sh

```diff
@@ -15,6 +15,20 @@ if [ -z "${LIBGUESTFS_MEMSIZE:-}" ]; then
     # only if we detect this case, but eh.
     export LIBGUESTFS_MEMSIZE=2048
 fi
+
+tmpd=$(mktemp -tdp /tmp gf-vsmt.XXXXXX)
```

Comment:
Can we only do this when on ppc64le?

So concretely, you'd move this down to within the if-statement block below, and then in `_coreos_gf_cleanup`, we'd do something like `if [ -n "${tmp_qemu_wrapper:-}" ]; then rm -rf "${tmp_qemu_wrapper}"; fi`.